.eslintrc.json

```diff
@@ -47,7 +47,17 @@
     "max-nested-callbacks": ["error", 5],
     "max-params": ["error", 4],
     "complexity": ["error", 20],
-    "@cspell/spellchecker": ["error"]
+    "@cspell/spellchecker": ["error"],
+    "import/no-extraneous-dependencies": [
+      "error",
+      {
+        "devDependencies": [
+          "**/*.stories.*",
```

Comment:
My initial regex wasn't inclusive enough. Can we do `"**/stories/**/*.*"` here and remove `import/no-extraneous-dependencies` from the files?